tex/alg-NT/norm-trace.tex

```diff
@@ -403,12 +429,36 @@ \section{On monogenic extensions}
 	\ii the cyclotomic field in \Cref{prob:ring_int_cyclotomic}.
 \end{itemize}
 Unfortunately, it is not true in general:
-the first counterexample is $\QQ(\alpha)$ for $\alpha$ a root of $X^3-X^2-2X-8$.
+the first counterexample is $K=\QQ(\alpha)$ for $\alpha$ a root of $X^3-X^2-2X-8$.
 
 We call an extension with this nice property \vocab{monogenic}.
 As we'll later see, monogenic extensions have a really nice factoring algorithm,
 \Cref{thm:factor_alg}.
 
+\begin{remark}
+	[What went wrong with $\OO_K$?]
+	As we have just mentioned above, as an abelian group, $\OO_K \cong \ZZ^3$,
+	so it's generated by finitely many elements.
+
+	In fact, $\{ 1, \alpha, \beta \}$ is a basis of $\OO_K$,
+	where $\beta=\frac{\alpha+\alpha^2}{2}$.
+	The group generated by $\{ 1, \alpha, \alpha^2 \}$ has index 2 in $\OO_K$ --
+	that is, $|\OO_K/\langle 1, \alpha, \alpha^2\rangle|=2$, and we misses
+	$\beta$.
+
+	If we try to pick $\{ 1, \beta, \beta^2 \}$ as a basis instead, again we
+	get $|\OO_K/\langle 1, \beta, \beta^2\rangle|=2$, and we misses $\alpha$.
+	If you explicitly compute it out, you can get
+	$\beta^2 = \frac{3\alpha^2 + 7\alpha}{2} + 6 = 3 \beta+2 \alpha+6$.
+
+	While this is not a proof that the extension is not monogenic, hopefully it
+	gives you a feeling of the structure of $\OO_K$.
+
+	% TODO is it true that, there exist some \alpha and integers c_2,
+	% \dots, c_{n-1} such that \OO_K is generated by {1, \alpha, \alpha^2/c_2,
+	% \dots, \alpha^{n-1}/c_{n-1}}?
```

Comment:
Try to give some intuition. (maybe if this part is true it would give more intuition -- 𝒪_K can be "visualized" as ℤ^n, but I'm not sure)